### DISCUSSION TITLE
Colored query output

### DISCUSSION THREAD

**Discussion by dominic-lcw** [duckdb/duckdb#15342] [Ideas]
I am thinking to add colored output in duckdb shell/jupyter notebook based on data types. I enjoyed this feature a lot in DBeaver.

Wondering if this is a good idea? Or should it be in an new (or existing) extension?

**Comment by drtdev1** [comment]
can you please tell/show how or what were you doing in DBeaver? thanks!

**Comment by Jalaj2508** [comment]
This can make the outputs and working more interactive, can be considered a good idea according to me.
